ovs/lib/storagerouter.py

```diff
@@ -1264,6 +1265,128 @@ def get_logfiles(local_storagerouter_guid):
+        :rtype: str
+        """
+        # Validations
+        required_params = {'port': (int, {'min': 1, 'max': 65535}),
+                           'vpool_guid': (str, Toolbox.regex_guid),
+                           'fragment_cache_on_read': (bool, None),
+                           'fragment_cache_on_write': (bool, None)}
+        Toolbox.verify_required_params(actual_params=parameters,
+                                       required_params=required_params)
+        vpool = VPool(parameters['vpool_guid'])
+        config_path = None
+        storagerouter = StorageRouter(storagerouter_guid)
+        local_storagerouter = StorageRouter(local_storagerouter_guid)
+        for sd in vpool.storagedrivers:
+            if sd.storagerouter_guid == storagerouter.guid:
+                if len(sd.alba_proxies) == 0:
+                    raise ValueError('No ALBA proxies configured for vPool {0} on StorageRouter {1}'.format(vpool.name, storagerouter.name))
+                config_path = '/ovs/vpools/{0}/proxies/{1}/config/{{0}}'.format(vpool.guid, sd.alba_proxies[0].guid)
+
+        if config_path is None:
+            raise ValueError('vPool {0} has not been extended to StorageRouter {1}'.format(vpool.name, storagerouter.name))
+
+        alba_backend_name = None
+        fragment_cache_on_read = parameters['fragment_cache_on_read']
+        fragment_cache_on_write = parameters['fragment_cache_on_write']
+        if fragment_cache_on_read is True or fragment_cache_on_write is True:
+            if 'path' in parameters:  # Local caching
+                required_params.update({'path': (str, re.compile('^/$|/.*[^.]$')),
+                                        'size': (int, {'min': 1, 'max': 10 * 1024})})
+                Toolbox.verify_required_params(actual_params=parameters,
+                                               required_params=required_params)
+            else:  # Accelerated ALBA
+                required_params.update({'backend_info': (dict, {'preset': (str, Toolbox.regex_preset),
+                                                                'alba_backend_name': (str, Toolbox.regex_backend)}),
+                                        'connection_info': (dict, {'host': (str, Toolbox.regex_ip, False),
+                                                                   'port': (int, {'min': 1, 'max': 65535}, False),
+                                                                   'client_id': (str, Toolbox.regex_guid, False),
+                                                                   'client_secret': (str, None, False)})})
+                Toolbox.verify_required_params(actual_params=parameters,
+                                               required_params=required_params)
+                alba_backend_name = parameters['backend_info']['alba_backend_name']
+                if Configuration.dir_exists(key='/ovs/arakoon/{0}-abm'.format(alba_backend_name)) is False:
```

Comment:
If it's an external cluster, you'll have to use the API calls like we use in the `add_vpool` logic.